package.json

```diff
@@ -200,6 +200,7 @@
     "@babel/register": "^7.12.1",
     "@babel/runtime": "^7.12.1",
     "@babel/traverse": "^7.12.1",
+    "@danmarshall/deckgl-typings": "^4.9.21",
```

Comment:
FYI @Pessimistress is currently working on converting deck.gl to typescript, the intention is that the resulting types will replace Dan Marshall's typings.